.github/workflows/ci.yml

```diff
@@ -295,11 +219,7 @@ jobs:
         run: |
           # Version resolution: use provided
           if [ -n "${{ github.event.inputs.version }}" ]; then
-            if [[ "${{ github.event.inputs.version }}" != *-SNAPSHOT ]]; then
-              resolvedVersion="${{ github.event.inputs.version }}-SNAPSHOT"
-            else
-              resolvedVersion="${{ github.event.inputs.version }}"
-            fi
```

Comment:
not sure why we ever changed this in tbdex-rs, but I think we should keep this code as it was, the ensure we never accidentally publish a version which doesn't have `-SNAPSHOT` suffix. thoughts?

Reply:
added back in